src/Morpho.sol

```diff
@@ -153,14 +153,10 @@ contract Morpho is IMorphoStaticTyping {
         require(isLltvEnabled[marketParams.lltv], ErrorsLib.LLTV_NOT_ENABLED);
         require(market[id].lastUpdate == 0, ErrorsLib.MARKET_ALREADY_CREATED);
 
-        // Safe "unchecked" cast.
-        market[id].lastUpdate = uint128(block.timestamp);
```

Comment:
why did we removed that?

Reply:
it's done in accrue interest, I find it safer like that

Reply:
update, I'm re-adding it (the result is the same but finally I think that it's better like that). See what I wrote [here](https://github.com/morpho-org/morpho-blue/pull/642#issue-2034113101). wdyt?